modules/ServerAPI/src/server/CustomerSpecificServerFunctions.coffee

```diff
@@ -415,56 +416,37 @@ exports.getAllAuthorObjectsInternal = (callback) ->
 		callback statusCode, json
 
 exports.relocateEntityFile = (fileValue, entityCodePrefix, entityCode, callback) ->
+	console.log "relocateEntityFile"
 	config = require "#{ACAS_HOME}/conf/compiled/conf.js"
 	uploadsPath = serverUtilityFunctions.makeAbsolutePath config.all.server.datafiles.relative_path
-	oldPath = uploadsPath + fileValue.fileValue
 	relEntitiesFolder = serverUtilityFunctions.getRelativeFolderPathForPrefix(entityCodePrefix)
 	if relEntitiesFolder==null
 		callback false
 		return
 	relEntityFolder = relEntitiesFolder + entityCode + "/"
-	absEntitiesFolder = uploadsPath + relEntitiesFolder
-	absEntityFolder = uploadsPath + relEntityFolder
+	oldPath = uploadsPath + fileValue.fileValue
 	if fileValue.comments != undefined and fileValue.comments != null
-		newPath = absEntityFolder + fileValue.comments
+		newPath = relEntityFolder + fileValue.comments
 	else
-		newPath = absEntityFolder + fileValue.fileValue
+		newPath = relEntityFolder + fileValue.fileValue
+	
+	filesToMove = [{sourceLocation: oldPath, targetLocation: newPath}]
+	movedFileResponse = await fileServices.moveDataFilesInternal(filesToMove)
```

Comment:
Basically all the same logic but instead of doing all the moving here, call the "moveDataFilesInternal" method to move the files to their correct location (local or gcs).